File: WebAPI/MROS_API/Models/ProductCategory.cs
using System;
using System.Collections.Generic;
using System.ComponentModel.DataAnnotations;
using System.Linq;
using System.Threading.Tasks;

namespace MROS_API.Models
{
    public class ProductCategory
    {
        [Key]
        public int ID { get; set; }
        public string CatName { get; set; }
        public string CatIMG { get; set; }
        public Restaurant Restaurant { get; set; }
        public bool Status { get; set; }
        public int Counter { get; set; }
    }
}

File: iOS/MROS/MROS/models/ProductCategory.swift
//
//  ProductCategory.swift
//  MROS
//
//  Created by Tayyip Muslu  on 19.01.2020.
//  Copyright © 2020 MUSLUNET. All rights reserved.
//

import Foundation

struct ProductCategory : Codable{
    var id: Int?
    var catName: String?
    var catIMG: String?
    var restaurant: Restaurant?
    var status:Bool?
    var counter: Int?
}
